WasatchNET/Spectrometer.cs

```diff
@@ -1491,40 +1569,59 @@ public virtual float laserTemperatureDegC
 
                 double rawD = raw;
 
-                // should this be 2.468? (see Dash3/WasatchDevices/Stroker785L_LaserTempSetpoint.py)
-                double voltage = 2.5 * rawD / 4096;
-                double resistance = 21450.0 * voltage / (2.5 - voltage);
-                if (resistance <= 0)
+                if (isSiG)
                 {
-                    logger.error("laserTemperatureDegC.get: invalid resistance (raw {0:x4}, voltage {1}, resistance {2:f2} ohms)",
-                        raw, voltage, resistance);
-                    return 0;
+                    double[] coeffs = new double[] { 1.5712971947853123e+000,

```

Comment:
Please specify these are for the for the 220250 Rev4 MAX1978ETM-T, because they'll be probably be different for the 220251. Also, we should consider logic to handle this differently on XS with MML.

Reply:
comment posting shortly